activerecord/lib/active_record/serialization.rb

```diff
@@ -9,7 +9,7 @@ module Serialization
     end
 
     def serializable_hash(options = nil)
-      options = options.try(:clone) || {}
+      options = options.try(:dup) || {}
```

Comment:
@sgrif That would be more elegant. It would be nice to backport to 4.2 though, in which case we can't use named arguments (because of the minimum ruby version requirement), right?